docs/src/pages/docs/functions/logic-functions.md

```diff
@@ -293,6 +396,37 @@ function. Either the original or modified value will be returned depending on
 the result of the predicate. Check out [`when`](#when) for a negated version of
 this function.
 
+```javascript
+import unless from 'crocks/logic/unless'
+
+import constant from 'crocks/combinator/constant'
+import isString from 'crocks/predicates/isString'
+import flip from 'crocks/combinators/flip'
+
+// prod :: Number -> Number -> Number
+const prod = a => b => b * a
+
+// doubleUnless :: (a -> boolean) -> Number -> Number
+const doubleUnless = flip(unless, prod(2))
+
+doubleUnless(constant(true), 21)
+//=> 21
+
+doubleUnless(constant(false), 21)
+//=> 42
+
+// toString :: a -> String
+const toString = x => x.toString()
+
+// ensureAllString :: [ a ] -> [ String ]
+const ensureAllString = unless(isString, toString)
+
+const testData = [1,2,'3', 4, true, false, new Date()]
```

Comment:
Need some breathing room on this data structure and probably could be multi-line. Also needs a siggy:
```js
// testData :: [ * ]
const testData =
[ 1, 2, '3', 4, true, false, new Date()]
```